tests/integration-tests/processors/processors-test.js

```diff
@@ -65,7 +65,30 @@ describe('Processors api', function() {
                 should(processors.length).greaterThanOrEqual(101);
             });
             after(async function() {
-                // TODO: when DELETE /processors is implemented, use processorsInserted to empty the table.
+                const processorIds = processorsInserted.map(processor => processor.body.id);
+                processorIds.forEach(async (processorId) => {
+                    await requestSender.deleteProcessor(processorId);
+                });
+            });
+        });
+        describe('DELETE /v1/processors/{processor_id}', () => {
+            it('insert a feature and then delete it', async () => {
+                const processor = generateRawJSProcessor('some_id');
+                const insertResponse = await requestSender.createProcessor(processor, validHeaders);
+                should(insertResponse.statusCode).equal(201);
+                const processorId = insertResponse.body.id;
+                const deleteResponse = await requestSender.deleteProcessor(processorId);
+                should(deleteResponse.statusCode).equal(204);
```

Comment:
after getting successful delete, send a `GET /processors/${processorId}` and expect 404